docs/detections/rules-cross-cluster-search.asciidoc

```diff
@@ -9,9 +9,11 @@
 
 This section explains the general process for setting up cross-cluster search in detection rules. For specific instructions on each part of the process, refer to the linked documentation.
 
-. On the local cluster, {ref}/remote-clusters.html[set up a connection] to the remote cluster. Note the unique name that you give to the remote cluster, because you'll need to use it throughout this process.
+NOTE: This procedure uses TLS certificate authentication to add remote clusters. {stack} 8.10.0 introduces an alternate method using {ref}/remote-clusters-api-key.html[API key authentication], but it is not yet included in the following steps.
 
-. On both the local and remote clusters, {ref}/remote-clusters.html[create a
+. On the local cluster, {ref}/remote-clusters-cert.html#remote-clusters-connect-cert[set up a connection] to the remote cluster. Note the unique name that you give to the remote cluster, because you'll need to use it throughout this process.
```

Comment:
```suggestion
. On the local cluster, {ref}/remote-clusters-cert.html[set up a connection] to the remote cluster. Note the unique name that you give to the remote cluster, because you'll need to use it throughout this process.
```

Reply:
I would link to the top-level page for setting up remote clusters with certificates, otherwise users may skip the setting up trust part.

Reply:
@abdonpijpelink Good point, thanks. I was trying to avoid pushing the reader to another long page with another set of steps for them to follow, but it sounds like it's somewhat unavoidable. Maybe I can give some extra context to help the reader target what they really need on that other page:
```suggestion
. On the local cluster, {ref}/remote-clusters-cert.html[establish trust and set up a connection] to the remote cluster. Note the unique name that you give to the remote cluster, because you'll need to use it throughout this process.
```